app/templates/volunteer/volunteerProfile.html

```diff
@@ -67,8 +67,50 @@ <h2>Which programs do you want to receive information about?</h2>
       </div>
 
   </div>
+{% if g.current_user.isCeltsAdmin%}
+<!-- This makes sure that the CeltsAdmins can make someone program manger or not.  -->
+{% if user.isCeltsStudentStaff%}
+  <div class="row">
+    <h2>Which programs do you want to make {{user.firstName}} {{user.lastName}} manager for?</h2>
+    <input id="{{ user.username }}" hidden>
+    <div>
+      {% for program in programs %}
+        {% set status = 0 %}
+          {% if program.id in permissionPrograms %}
+            {% set status = 1 %}
+          {% else %}
+          {% endif %}
+              <input  type="checkbox" id="{{ program.id }}" {{ 'checked' if status==1 else 'unchecked' }}  onclick="updateManagers(this,{{user.username}},{{status}})">
+              <label  style="padding-right:50px;" for="{{ program.id }}">{{program.programName}}</label>
+      {% endfor %}
+    </div>
+  </div>
+  {% endif %}
+{% else %}
+<div class="row">
```

Comment:
There's a missing case here. We probably don't want to show this section at all if the user is a regular student (not staff)

Reply:
Fixed by adding another case.